ui/src/SearchResults.tsx

```diff
@@ -63,16 +74,33 @@ function SearchPage(props: any) {
         </div>
       );
     });
+  }
 
-    return (
-      <div className="search">
-        <NavBar></NavBar>
-        <header className="searchHeader">
-          <h1>Search Results</h1>
-          <ul>{memes}</ul>
-        </header>
-      </div>
-    );
+  let usernames;
+  if (usernamesResponse.length === 0) {
+    usernames = <div>No users exist with this search criteria</div>;
+  } else {
+    usernames = usernamesResponse.map(function (username) {
+      return (
+        <div>
+          {" "}
+          <h3>{username}</h3>
```

Comment:
H3 isn't really appropriate here.  It may style it the way you want, but the heading tags have semantic meaning as headers.  This isn't a header, it's data.